tests/components/cloud/test_binary_sensor.py

```diff
@@ -1,24 +1,23 @@
 """Tests for the cloud binary sensor."""
 from unittest.mock import Mock
 
+from asynctest import patch
+
 from homeassistant.components.cloud.const import DISPATCHER_REMOTE_UPDATE
 from homeassistant.setup import async_setup_component
 
 
 async def test_remote_connection_sensor(hass):
     """Test the remote connection sensor."""
-    from homeassistant.components.cloud import binary_sensor as bin_sensor
-
-    bin_sensor.WAIT_UNTIL_CHANGE = 0
-
     assert await async_setup_component(hass, "cloud", {"cloud": {}})
     await hass.async_block_till_done()
 
     assert hass.states.get("binary_sensor.remote_ui") is None
 
     # Fake connection/discovery
-    org_cloud = hass.data["cloud"]
-    await org_cloud.iot._on_connect[-1]()
+    await hass.helpers.discovery.async_load_platform(
```

Comment:
It would be nicer if we didn't have to mock integration code directly but could mock things in the periphery instead to have the platform load.

Reply:
I agree, our cloud mocks have not kept up over time.